### ISSUE TITLE
Getting invalid certificate error

### ISSUE THREAD

**Issue by sheraz1000** [IRSgov/IDES-Data-Preparation-Dot-Net#22]
I get the error for RC020, "Invalid Certificate" while uploading it to the ides test environment , I am using PFX certificate , any pointers why am I getting this issue

**Comment by DanielCrouch** [comment]
That error would be related to the public key that was used to enroll in IDES. Is it a valid organizational (not personal) certificate from one of the approved Certificate Authorities? If there is a problem with it, or if you think there is, I suggest contacting the IDES Help Desk folks at https://www.ides-support.com/Support. If there is an issue with an account or certificate, you can check with them.

**Comment by sheraz1000** [comment]
Yes,  I have used the certificate before ,but  i got it renewed this year, it is also a valid organisation certificate, i have no idea what issue could it be , i will contact IDES Help Desk.
